loader/src/main/scala/com/actian/spark_vector/loader/spark_sql_utils/package.scala

```diff
@@ -15,8 +15,13 @@
  */
 package com.actian.spark_vector.loader
 
-package object command {
+package object spark_sql_utils {
   def sparkQuote(name: String): String = s"`$name`"
 
-  def colsSelectStatement(cols: Option[Seq[String]]): String = cols.map(_.map(sparkQuote(_)).mkString(", ")).getOrElse("*")
+  def colsSelectStatement(cols: Option[Seq[String]]): String = {
```

Comment:
colsSelectStatement -> selectStatement.
Also, you can get rid of one pair of {}: def selectStatement(...) = cols match {
}